newIDE/app/src/InstancesEditor/SelectionRectangle.js

```diff
@@ -10,9 +11,10 @@ export default class SelectionRectangle {
   pixiRectangle: PIXI.Graphics;
   selectionRectangleStart: any;
   selectionRectangleEnd: any;
-  _instancesInSelectionRectangle: gdInitialInstance[];
+  _instancesInSelectionRectangle: any[];
```

Comment:
Is this to appease Flow? I think this should be kept to `gdInitialInstance[]`?

Reply:
I don't remember. I'm reverting it to check this next time I do a flow pass.